packages/eslint-config-custom/index.js

```diff
@@ -13,6 +13,17 @@ module.exports = {
     '@next/next/no-html-link-for-pages': 'off',
     '@typescript-eslint/no-non-null-assertion': 'off',
     '@typescript-eslint/no-confusing-void-expression': ['error', { ignoreArrowShorthand: true }],
+    'tailwindcss/no-custom-classname': [
+      'warn',
+      {
+        whitelist: [
+          'bg\\-(card-radial|charcoal|teal|accent|background|teal/80|destructive|destructive/90)',
+          'text\\-(muted-foreground|accent-foreground|primary-foreground|destructive-foreground|primary|teal)',
+          'ring\\-(offset-background|ring)',
+          'border\\-(input|teal)',
+        ],
+      },
+    ],
```

Comment:
Why is this necessary? This will be annoying to have to maintain.

Reply:
When I remove this, I do not get any warning

Reply:
By default, is `tailwindcss/no-custom-classname` off? When I remove these lines, I get no warnings. Are you adding in a new rule for extra protection?

Reply:
or do you not have a notification for this rule in your code? and a problem with my IDE